backends/rapidpro/backend.go

```diff
@@ -655,7 +649,7 @@ func (b *backend) SaveAttachment(ctx context.Context, ch courier.Channel, conten
 
 	orgID := ch.(*Channel).OrgID()
 
-	path := filepath.Join(b.config.S3AttachmentsPrefix, strconv.FormatInt(int64(orgID), 10), filename[:4], filename[4:8], filename)
+	path := filepath.Join("attachments", strconv.FormatInt(int64(orgID), 10), filename[:4], filename[4:8], filename)
```

Comment:
Why not keep that as `media`?
Are the files actually moved on that path now?

Reply:
This is just what is set on the default config (used by tests) that we always overwrite on prod to attachments. RP, mailroom and courier should all use the same `attachments` prefix.